tests/backup_restore.rs

```diff
@@ -7,49 +7,39 @@
 //! You can run them with 'nextest':
 //! `cargo nextest run -E 'test(backup)'`.
 
-use abscissa_core::testing::prelude::CmdRunner;
-use aho_corasick::PatternID;
 use dircmp::Comparison;
-use pretty_assertions::assert_eq;
-use std::io::Read;
 use tempfile::{tempdir, TempDir};
 
-use rustic_testing::{get_matches, TestResult};
+use assert_cmd::Command;
+use predicates::prelude::{predicate, PredicateBooleanExt};
 
-pub fn rustic_runner(temp_dir: &TempDir) -> CmdRunner {
+use rustic_testing::TestResult;
+
+pub fn rustic_runner(temp_dir: &TempDir) -> TestResult<Command> {
     let password = "test";
     let repo_dir = temp_dir.path().join("repo");
-    let mut runner = CmdRunner::new(env!("CARGO_BIN_EXE_rustic"));
+
+    let mut runner = Command::new(env!("CARGO_BIN_EXE_rustic"));
+
     runner
         .arg("-r")
         .arg(repo_dir)
         .arg("--password")
         .arg(password)
-        .arg("--no-progress")
-        .capture_stdout()
-        .capture_stderr();
-    runner
+        .arg("--no-progress");
+
+    Ok(runner)
 }
 
 fn setup() -> TestResult<TempDir> {
     let temp_dir = tempdir()?;
-    let mut runner = rustic_runner(&temp_dir);
-    let mut cmd = runner.args(["init"]).run();
-
-    let mut stdout = String::new();
-    let mut stderr = String::new();
-    cmd.stdout().read_to_string(&mut stdout)?;
-    cmd.stderr().read_to_string(&mut stderr)?;
-
-    let patterns = &["successfully added.", "successfully created."];
-    let matches = get_matches(patterns, stderr)?;
-
-    assert_eq!(
-        matches,
-        vec![(PatternID::must(0), 19), (PatternID::must(1), 21),]
-    );
+    rustic_runner(&temp_dir)?
+        .args(["init"])
+        .assert()
+        .success()
+        .stderr(predicate::str::contains("successfully created."))
+        .stderr(predicate::str::contains("successfully added."));
```

Comment:
Is that intentional, that we log that on stderr on init?

Reply:
Yes, it is all an `info!` log here. And logs are by default printed to stderr. You could even turn this message off by using `--log-level=warn/error/off`.